### DISCUSSION TITLE
Purpose of moon run --force?

### DISCUSSION THREAD

**Discussion by dudicoco** [moonrepo/moon#1723] [Q&A] [answered]
Hi,

The [`moon run --force` flag](https://moonrepo.dev/docs/commands/run#options) description states:
> -f, --force - Force run and ignore touched files and affected status. Will not query VCS.

How is that different than just running `moon run` without the `--affected` flag?

**Comment by milesj** [comment]
It's a bit confusing and this will change in the future, but how touched files is determined changes based on CI vs local and the `--remote` argument. Touched files are always gathered regardless of `--affected`, but checking if tasks are affected only happens if the latter argument is provided.

`--force` just bypasses a lot of this.

**Reply by dudicoco** [reply]
@milesj can you describe a use case where running `moon run` with and without the `--force` flag would make a difference?

**Reply by milesj** [reply, marked as answer]
It pretty much does nothing at this point. Before it would avoid affected + touched files, but we changed it to always loading touched files.
